packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -238,6 +238,15 @@ export class ListItem
    */
   @Event({ cancelable: false }) calciteInternalListItemSelect: EventEmitter<void>;
 
+  /**
+   *
+   * @internal
+   */
+  @Event({ cancelable: false })
+  calciteInternalListItemSelectMultiple: EventEmitter<{
```

Comment:
Nitpick: I think the event name or the detail prop can be renamed to avoid some redundancy.

Reply:
Yeah I couldn't think of anything better. Since its internal i'll leave as is.